### DISCUSSION TITLE
how to air-gap install rke2 with calico

### DISCUSSION THREAD

**Discussion by fairmonk** [rancher/rke2#3758] [Q&A] [answered]
managed to install rke2 in air-gapped environment with default CANAL CNI plugin, using tarball merhod ( reference here -> https://docs.rke2.io/install/airgap )
however I need CALICO CNI plugin.
how can I install with default plugin calico ? haven't found any cni references inside install script here -> https://get.rke2.io/
Any response appreciated.

**Comment by brandond** [comment, marked as answer]
The install script does not configure anything for you, nor does it start RKE2 at the end of the install process. This is so that you can configure RKE2 before starting it.

You should follow the airgap installation instructions, including downloading the correct tarballs for your selected CNI. After installation, set the CNI in the configuration file before starting RKE2 for the first time.

That's it.
